crates/libcontainer/src/tty.rs

```diff
@@ -165,84 +165,72 @@ fn connect_stdio(stdin: &RawFd, stdout: &RawFd, stderr: &RawFd) -> Result<()> {
 mod tests {
     use super::*;
 
-    use anyhow::Result;
+    use anyhow::{Ok, Result};
     use serial_test::serial;
-    use std::env;
-    use std::fs::{self, File};
+    use std::fs::File;
     use std::os::unix::net::UnixListener;
-    use std::path::PathBuf;
 
     const CONSOLE_SOCKET: &str = "console-socket";
 
-    fn setup() -> Result<(tempfile::TempDir, PathBuf, PathBuf)> {
-        let testdir = tempfile::tempdir()?;
-        let rundir_path = Path::join(testdir.path(), "run");
-        fs::create_dir(&rundir_path)?;
-        let socket_path = Path::new(&rundir_path).join("socket");
-        let _ = File::create(&socket_path);
-        env::set_current_dir(&testdir)?;
-        Ok((testdir, rundir_path, socket_path))
-    }
-
```

Comment:
Hey, what is the reason behind changing this logic? I'm not saying changing it is wrong, just not sure.

Reply:
Just refactoring. I don't intend to change the core logic of these tests.